lib/di/interceptor/app_interceptor.dart

```diff
@@ -1,3 +1,4 @@
+// ignore: unused_import
```

Comment:
What does this mean? Do we need to let the linter skip checking import here?

Reply:
Yes, correct. 

It's related to the comment below 🙏 

I've updated the insights 🚀